### PR TITLE
feat: cleanup org

### LINKED ISSUE #255: Remove the organization section
As decision from an [internal discussion](https://github.com/orgs/fastify/discussions/48), the fastify maintainers do agree to do the following:

```[tasklist]
### Tasks
- [ ] Remove the `using` section from the homepage
- [ ] Remove from the [org page](https://fastify.dev/organisations/) all the companies that are not sponsor or are not affiliated to any fastify's maintainers (regardless the team)
```

## Task 1

This `Using` section and keep the `Sponsors` only:

image

Add anyway the link to the `https://fastify.dev/organisations/` page.

## Task 2

The page https://fastify.dev/organisations/ should be cleaned by external orgs.
We must replace the `How to be featured here` section with a call to action to the sponsor page:
https://github.com/sponsors/fastify

### PR DESCRIPTION
**Pull request by Eomm**
Closes https://github.com/fastify/website/issues/255

I left the member's org I know - so be sure your @fastify/collaborators is included/added as you wish

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by jsumners — APPROVED** [review]

**Review by gurgunday — APPROVED** [review]

**Review by Fdawgs — APPROVED** [review]

**Review by dancastillo — APPROVED** [review]

File: src/components/Organisations/index.jsx [hunk 1]
```diff
@@ -6,9 +6,12 @@ import organizationsData from '@site/static/generated/organisations.json'
...
 export default function Organisations({ maxItems, onlySponsors }) {
-  const orgs = onlySponsors //
-    ? organizationsData.filter((org) => org.sponsor)
-    : organizationsData
+  const orgs =
+    onlySponsors === 'true' //
+      ? organizationsData.filter((org) => org.sponsor)
+      : onlySponsors === 'false' //
+        ? organizationsData.filter((org) => !org.sponsor)
+        : organizationsData
```
Comment (Uzlopak) [inline src/components/Organisations/index.jsx]: if onlySponsors is false we filter all sponsors?

Comment (Eomm) [reply src/components/Organisations/index.jsx]: you are right - I have been lazy, will clean it

image

Comment (Eomm) [reply src/components/Organisations/index.jsx]: Done @Uzlopak

**Review by Uzlopak — APPROVED** [review]
LGTM

**Pull request merged**